superset/views/core.py

```diff
@@ -313,9 +321,21 @@ def form_get(self, form):
         form.skip_blank_lines.data = True
         form.infer_datetime_format.data = True
         form.decimal.data = '.'
-        form.if_exists.data = 'append'
+        form.if_exists.data = 'fail'
 
     def form_post(self, form):
+        database = form.con.data
+        schema_name = form.schema.data or ''
+
+        if not self.is_schema_allowed(database, schema_name):
+            message = _('Database "{0}" Schema "{1}" is not allowed for csv uploads. '
+                        'Please contact Superset Admin'.format(database.database_name,
+                                                               schema_name))
+            flash(
```

Comment:
This can be written in a single language:

```python
flash(message, 'danger')
```

Reply:
woops. I must have copy-paste from somewhere and then made some changes but did not aware of the styling issue. thanks